lib/Installation/ModuleRegistration/ModuleRegistrationController.pm

```diff
@@ -63,4 +63,9 @@ sub get_modules {
     return $self->get_module_registration_page()->get_modules();
 }
 
+sub uncheck_hide_development_versions {
```

Comment:
you already have that method above `view_development_versions`.

Reply:
Ok, use the existing method.